### PR TITLE
Bugfix - Zentaro

### PR DESCRIPTION
**Pull request by jamesoddy**

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
What's the reasoning for removing `/cancel-prompt`?

**Comment by jamesoddy** [comment]
"/cancel-prompt is a legacy command that isn't really supported any more"
"manual mode to cancel the prompt should cancel the prompt in a much more elegant way"

Essentially, people should not be using cancel-prompt so it seems best to take it out of the system as it may be causing a number of crashes on the server.

**Comment by NoahTheDuke** [comment]
Cool, that makes total sense. Thanks!

File: server/game/PlayDisguisedCharacterAction.js [hunk 1]
```diff
@@ -48,6 +49,9 @@ class DisguisedReduceableFateCost extends ReduceableFateCost {
     }
 
     getReducedCost(context) {
+        if(!context.costs.chooseDisguisedCharacter) {
+            return;
```
Comment (jeremylarner) [inline server/game/PlayDisguisedCharacterAction.js]: I'm not sure what this is intended to achieve - you're just pushing a potential crash somewhere else...

**Pull request merged**